testsuite/integration/basic/src/test/java/org/jboss/as/test/integration/jpa/epcpropagation/unsync/BMTEPCStatefulBean.java

```diff
@@ -84,7 +84,7 @@ public void allowjoinedunsync() throws SystemException, NotSupportedException, H
         UserTransaction userTxn = sessionContext.getUserTransaction();
         userTxn.begin();
         try {
-            //Join Transaction to force unsynchronized persistence context to be associated with jta tx
+            //Join Transaction to force unsynchronized persistence context to be associated with Jakarta Transactions tx
```

Comment:
`Jakarta Transactions tx` -> `the Jakarta Transaction`

Reply:
I disagree as I don't think a "Jakarta Transaction" is a defined thing. I would say either the "Jakarta Transactions tx" or maybe just "transaction"

Reply:
or "Jakarta Transactions transaction"

Reply:
Okay, I'm putting `Jakarta Transactions tx`

Reply:
Please apply the style choice from previous comments (either use `(Jakarta Transactions) tx` or `Jakarta Transactions tx`.